0001-kaa-protocol/README.md

```diff
@@ -1,9 +1,9 @@
 ---
-name: Kaa Protocol
+name: Kaa protocol
 shortname: 1/KP
 status: draft
-editor: Alexey Shmalko <ashmalko@cybervisiontech.com>
-contributors: Alexey Gamov <agamov@cybervisiontech.com>, Andrew Kokhanovskyi <akokhanovskyi@cybervisiontech.com>
+editor: Alexey Shmalko<ashmalko@cybervisiontech.com>
+contributors: Alexey Gamov<agamov@cybervisiontech.com>, Andrew Kokhanovskyi<akokhanovskyi@cybervisiontech.com>
```

Comment:
I don't see how it's a GitHub rendering issue -- the page renders fine with the spaces. But if you say we're going to use the RFCs in other software environments -- ok, bringing back the spaces.